atomic_reactor/utils/operator.py

```diff
@@ -304,11 +486,7 @@ def __init__(self, path, data, pullspec_heuristic=default_pullspec_heuristic):
             implementation will be used. For more information, see the
             default_pullspec_heuristic() function in this module.
         """
-        try:
-            if data.get("kind") != OPERATOR_CSV_KIND:
-                raise NotOperatorCSV("Not a ClusterServiceVersion")
-        except AttributeError as exc:
-            raise NotOperatorCSV("File does not contain a YAML object") from exc
+        check_csv(data, _mini_csv_schema)
```

Comment:
What's the opinion to separate these lines into a function?

The schema is defined in module level, is it necessary to pass into the function via a parameter? Just reference it directly in that function?

Reply:
@tkdchen; You're abslutely right, it should probably be a class method. IIRC, my goal was to simplify `OperatorCSV.__init__()`.

Reply:
@tkdchen;
I'm not smart enough to unwind how to bypass schema checking in something like e.g. `utils/test_operator.py::test_get_pullspecs_some_locations()`.

Having `check_csv()` as a module function (it doesn't access cls or self) means I can easily mock it out of the unit tests.

Good catch on the arg :)